src/utils.js

```diff
@@ -219,3 +218,10 @@ export function update(
 
   return set(state, path, next)
 }
+
+/**
+ * A couple of methods use identity functions. This avoids duplication.
+ */
+export function identity<T>(n: T): T {
```

Comment:
`<T>` is what Flow uses in their examples, but this could be clearer. Any thoughts? maybe `<Type>`?